src/watcher.js

```diff
@@ -55,6 +55,10 @@ Watcher.prototype = {
             }
           });
         });
+
+        streams[bundleFile].on('error', function (error) {
+          that._events.emit('error', error, bundleFile);
+        });
```

Comment:
Also, I've made the `error` object the first arg 'cause I think that is what people would generally expect from that callback.